Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -160,7 +160,9 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// When multiple references resolve to the same documentation, use this to find the main reference that is associated with the node in the topic graph.
     ///
     /// The key to lookup the main reference is the alias reference. A main reference can have many aliases but an alias can only have one main reference.
-    var referenceAliases = [ResolvedTopicReference: ResolvedTopicReference]()
+    private var referenceAliases = [ResolvedTopicReference: ResolvedTopicReference]()
+    
+    private var canonicalReferences = [ResolvedTopicReference: Set<ResolvedTopicReference>]()
```

Comment:
I think I'm to blame for these names but I think that they're backwards:
- `referenceAliases` is a mapping from alias references (the key) to the canonical reference (the value)/
- `canonicalReferences` is a mapping from a canonical reference (the key) to all the non-canonical aliases (the value).